eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/service/nonhuman/dbsnp/DbsnpClusteredVariantMonotonicAccessioningService.java

```diff
@@ -53,4 +55,13 @@ public String getHash(IClusteredVariant variant) {
     public List<AccessionWrapper<IClusteredVariant, String, Long>> getByHash(List<String> hashes) {
```

Comment:
can't you use the function in the parent? you deleted this function from the Submitted classes.

Reply:
Thanks, I missed that